core/shared/src/main/scala/zio/Schedule.scala

```diff
@@ -1126,6 +1127,34 @@ object Schedule {
     Schedule(loop(None, 0L))
   }
 
+  /**
+   * Cron-like schedule that recurs every specified `minute` of each hour.
+   * Exact second of the minute can be provided as optional `second` param.
+   * Producing a count of repeats: 0, 1, 2.
+   */
+  def minuteOfHour(minute: Int, second: Int = 0): Schedule[Any, Any, Long] = {
```

Comment:
We should delete `second` and instead provide `secondOfMinute` as a separate constructor.